### ISSUE TITLE
NodeJS: Error: ClientTask error: Client(SyncedNodePoolEmpty) when doing any operations with a client-instance

### ISSUE THREAD

**Issue by 0xrubes** [iotaledger-archive/iota.rs#502]
## Bug description

For some reason, after initialization with given nodes to connect to, the node-pool seems to be empty, causing any operation on an client-instance to fail with `Error: ClientTask error: Client(SyncedNodePoolEmpty)`. Same for the examples in the nodejs-docs.
A few days ago, this worked for me.

## Versions

Which version of Node are you running?

- v12.13.0

Which version of Rust are you running?

-  rustc 1.48.0 (7eac88abb 2020-11-16) 

Which version of the library are you using?

- @iota/client npm package 0.2.0

## Hardware specification

What hardware are you using?

- Operating system: Windows 10

## Steps To reproduce the bug

Explain how the maintainer can reproduce the bug.

1.  Build a client-instance
2.  Execute any operation on it, such as `getInfo()` or send a message

Basically this:
```
const { ClientBuilder } = require("@iota/client");

const nodeArray = [
    "https://api.hornet-0.testnet.chrysalis2.com",
    "https://api.hornet-1.testnet.chrysalis2.com",
    "https://api.hornet-2.testnet.chrysalis2.com",
    "https://api.hornet-3.testnet.schrysalis2.com",
]

const client = new ClientBuilder()
    // .node('https://api.lb-0.testnet.chrysalis2.com') //doesnt work either
    .nodes(nodeArray)
    .build()

client.getInfo()
    .then(info => console.log(info))
    .catch(e => console.log("Error getInfo(): \n" + e))

```

## Expected behaviour

Successfully execute operations on client-instance

## Actual behaviour

Throws an error.

## Errors

`Error: ClientTask error: Client(SyncedNodePoolEmpty)  `

**Comment by Thoralf-M** [comment]
The npm package is outdated, the nodeinfo got changed, but we have already `0.3.1` published where it should work fine

**Comment by 0xrubes** [comment]
Ahh you are absolutely right, that was the issue. Thanks!
